src/main/java/org/jitsi/videobridge/Conference.java

```diff
@@ -435,9 +436,47 @@ void dominantSpeakerChanged()
             broadcastMessage(
                     createDominantSpeakerEndpointChangeEvent(
                         dominantSpeaker.getID()));
+            if (getEndpointCount() >= 3)
+            {
+                double senderRtt = getRtt(dominantSpeaker);
+                double maxReceiverRtt = getMaxReceiverRtt(dominantSpeaker.getID());
+                // We add an additional 10ms delay to reduce the risk of the keyframe arriving
+                // too early
+                double keyframeDelay = maxReceiverRtt + senderRtt + 10;
```

Comment:
This should subtract the sender RTT: https://github.com/jitsi/jitsi-videobridge/blob/master/src/main/java/org/jitsi/videobridge/VideoChannel.java#L640

Reply:
nice catch